### ISSUE TITLE
[BUG] : Bs manager will not let me download any bs versions

### ISSUE THREAD

**Issue by SunnyandRadium** [Zagrios/bs-manager#420]

**Comment by Zagrios** [comment]
Hi, can you send your logs ?
Go to settings -> scroll down -> click on "open logs"

**Comment by CallMeMSL** [comment]
Hi, I have the same issue,

These are my logs:

```
[2024-02-22 22:32:50.422] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:32:50.427] [info]  [OUTPUT] isLivInstalled() false
[2024-02-22 22:32:50.427] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:32:50.428] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:32:50.921] [info]  Checking for update
[2024-02-22 22:32:50.940] [info]  Generated new staging user ID: 45c8eb60-d980-5622-a9bc-fee3084ea272
[2024-02-22 22:32:52.258] [info]  Update for version 1.4.5 is not available (latest version: 1.4.5, downgrade is disallowed).
[2024-02-22 22:32:52.584] [info]  getVersionOfBSFolder E:\Steam\steamapps\common\Beat Saber { steam: true }
[2024-02-22 22:32:52.590] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:32:52.590] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:36:06.704] [info]  getVersionOfBSFolder E:\Steam\steamapps\common\Beat Saber { steam: true }
[2024-02-22 22:36:06.713] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found
[2024-02-22 22:36:06.714] [info]  Registry key "HKCU\SOFTWARE\Oculus VR, LLC\Oculus\Libraries" not found

```

**Comment by SunnyandRadium** [comment]
Here is my logs 

[2024-02-22 16:41:10.726] [info]  [OUTPUT] isLivInstalled() true
[2024-02-22 16:41:10.731] [error] Unhandled Exception UnhandledRejection Error: ENOENT: no such file or directory, mkdir 'E:\BS manager\BSManager'
    at mkdirSync (node:fs:1398:3)
    at Object.mkdirSync (node:electron/js2c/asar_bundle:2:12611)
    at e.exports._ensureDirectory (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:137978)
    at e.exports.get store [as store] (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:136719)
    at new N (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:134359)
    at new e.exports (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:14:72899)
    at o.initStore (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:14:1032140)
[2024-02-22 16:41:11.217] [info]  Checking for update
[2024-02-22 16:41:12.202] [info]  Update for version 1.4.5 is not available (latest version: 1.4.5, downgrade is disallowed).
[2024-02-22 16:41:12.646] [info]  getVersionOfBSFolder C:\Program Files (x86)\Steam\steamapps\common\Beat Saber { steam: true }

**Comment by SunnyandRadium** [comment]
Here is my log

[2024-02-22 16:41:10.726] [info]  [OUTPUT] isLivInstalled() true
[2024-02-22 16:41:10.731] [error] Unhandled Exception UnhandledRejection Error: ENOENT: no such file or directory, mkdir 'E:\BS manager\BSManager'
    at mkdirSync (node:fs:1398:3)
    at Object.mkdirSync (node:electron/js2c/asar_bundle:2:12611)
    at e.exports._ensureDirectory (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:137978)
    at e.exports.get store [as store] (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:136719)
    at new N (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:8:134359)
    at new e.exports (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:14:72899)
    at o.initStore (C:\Users\12394\AppData\Local\Programs\bs-manager\resources\app.asar\dist\main\main.js:14:1032140)
[2024-02-22 16:41:11.217] [info]  Checking for update
[2024-02-22 16:41:12.202] [info]  Update for version 1.4.5 is not available (latest version: 1.4.5, downgrade is disallowed).
[2024-02-22 16:41:12.646] [info]  getVersionOfBSFolder C:\Program Files (x86)\Steam\steamapps\common\Beat Saber { steam: true }

**Comment by SunnyandRadium** [comment]
And this if you need it to fix the issue

[2023-11-25 14:17:36.804] [info]  DepotDownloader process end with code 0
[2023-11-25 14:17:36.808] [info]  try get version from folder E:\BS manager\BSManager\BSInstances\1.29.1
[2023-11-25 14:52:07.656] [info]  BS process exit code -1
[2023-11-25 18:33:24.612] [info]  Checking for update
[2023-11-25 18:33:25.832] [info]  Update for version 1.3.2 is not available (latest version: 1.3.2, downgrade is disallowed).
[2023-11-25 18:33:26.705] [info]  try get version from folder E:\BS manager\BSManager\BSInstances\1.29.1
[2023-11-25 18:34:01.037] [info]  BS process exit code -1
[2023-11-25 19:16:20.231] [info]  BS process exit code -1
[2023-11-25 21:46:07.715] [info]  Checking for update
[2023-11-25 21:46:08.736] [info]  Update for version 1.3.2 is not available (latest version: 1.3.2, downgrade is disallowed).
[2023-11-25 21:46:09.364] [info]  try get version from folder E:\BS manager\BSManager\BSInstances\1.29.1
[2023-11-25 21:48:24.121] [info]  BS process exit code -1
[2023-11-25 23:10:02.896] [info]  Checking for update
[2023-11-25 23:10:04.201] [info]  Update for version 1.3.2 is not available (latest version: 1.3.2, downgrade is disallowed).
[2023-11-25 23:10:05.099] [info]  try get version from folder E:\BS manager\BSManager\BSInstances\1.29.1
[2023-11-25 23:18:47.680] [info]  BS process exit code -1
